tools/scoring/tests/samples/proposed/entity_connection_identification.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is used as an example of a valid building configuration with
```

Comment:
can you use the new format here?
can we reuse other files?

Reply:
https://github.com/google/digitalbuildings/pull/527/commits/8dab8560e401d283009391f1a86d22d6ce668076

I find reusing the other files to be extremely confusing.

Reply:
Found what appears to be an instance validator bug. If a config yaml contains a single entity in the old format, `validator.handler.Deserialize()` outputs as expected:

```({'90ebdd30-efdf-4aec-82d7-90f8e03a6792': <validate.entity_instance.EntityInstance object at 0x7f8b6e066cd0>}, <ConfigMode.INITIALIZE: 'INITIALIZE'>)```

If the same entity is passed in the new format, however:

```({}, <ConfigMode.INITIALIZE: 'INITIALIZE'>)```

This does not occur if there is more than one entity in the config.

Reply:
https://github.com/google/digitalbuildings/issues/530